### PR TITLE
[14.0][IMP] stock_cycle_count: Add responsible_id to stock.inventory, linked with cycle count

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
This module adds responsible in stock.inventory and inherits from stock.cycle.count. Not added the field in stock.inventory because it disappears in 15.0.

@ForgeFlow

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

File: stock_cycle_count/models/stock_inventory.py [hunk 1]
```diff
@@ -36,6 +36,12 @@ def _compute_inventory_accuracy(self):
         store=True,
         group_operator="avg",
     )
+    responsible_id = fields.Many2one(
+        comodel_name="res.users",
+        string="Responsible Person",
```
Comment (LoisRForgeFlow) [inline stock_cycle_count/models/stock_inventory.py]: just responsible

Comment (DavidJForgeFlow) [reply stock_cycle_count/models/stock_inventory.py]: Done!

File: stock_cycle_count/models/stock_cycle_count.py [hunk 2]
```diff
@@ -95,6 +95,7 @@ def action_create_inventory_adjustment(self):
         for rec in self:
             data = rec._prepare_inventory_adjustment()
```
Comment (LoisRForgeFlow) [inline stock_cycle_count/models/stock_cycle_count.py]: fill the reponsible in the hook like all the other data.

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM now

**Pull request closed without merging**